core/src/main/java/apoc/export/util/ExportConfig.java

```diff
@@ -217,4 +221,12 @@ public boolean isSampling() {
     public boolean ifNotExists() {
         return ifNotExists;
     }
+
+    public boolean isSaveIndexNames() {
```

Comment:
Nitpick: `shouldSaveIndexNames` and `shouldSaveContraintNames` maybe as better names?